python/CMakeLists.txt

```diff
@@ -112,6 +115,9 @@ add_mlir_python_modules(GcPythonModules
     GcPythonSources
     MLIRPythonExtension.RegisterEverything
     MLIRPythonSources
+    MLIRPythonSources.ExecutionEngine
   COMMON_CAPI_LINK_LIBS
     GcPythonCAPI
-  )
\ No newline at end of file
+  )
+
+configure_file(config.py.in ${MLIR_BINARY_DIR}/python_packages/gc_mlir_core/gc_mlir/config.py @ONLY)
```

Comment:
you can test to print `message(STATUS "${MLIR_RUNNER_UTILS_DIR}")` to check if `MLIR_RUNNER_UTILS_DIR` is correctly defined here.

Reply:
> you can test to print `message(STATUS "${MLIR_RUNNER_UTILS_DIR}")` to check if `MLIR_RUNNER_UTILS_DIR` is correctly defined here.

I tried before and the value of `${MLIR_RUNNER_UTILS_DIR}` is empty here, it was set in `graph-compiler/test/mlir/test/CMakeLists.txt`

Reply:
OK, I see. No further questions here.